src/JsCallback.php

```diff
@@ -115,12 +115,8 @@ public function set($fx = null, $args = null)
                 $ajaxec = $response ? $this->getAjaxec($response, $chain) : null;
 
                 $this->terminateAjax($ajaxec);
-            } catch (\Atk4\Data\ValidationException $e) {
-                // Validation exceptions will be presented to user in a friendly way
-                $msg = new Message($e->getMessage());
-                $msg->addClass('error');
-
-                $this->terminateAjax(null, $msg->getHtml(), false);
+            } catch (\Exception $e) {
+                $this->terminateAjax(null, $this->getApp()->renderExceptionHtml($e), false);
```

Comment:
one question - why catch, when we can leave to app exception handler?

Reply:
Do you mean `throw ($e)`?

Reply:
or remove entire try - catch block?

Reply:
remove entire catch, I think it is not needed as renderExceptionHtml is the normal exception render when an exception is caught in app, but of course check if the behaviour is the same...

Reply:
Done!